lib/tree-view.coffee

```diff
@@ -755,3 +784,49 @@ class TreeView extends View
   # Returns boolean
   multiSelectEnabled: ->
     @list[0].classList.contains('multi-select')
+
+  onDragEnter: (e) ->
+    e.stopPropagation()
+
+    e.target.classList.add('dragenter')
+
+  onDragLeave: (e) ->
+    e.stopPropagation()
+
+    e.target.classList.remove('dragenter')
+
+  # Public: Handle entry name object dragstart event
+  #
+  # returns noop
```

Comment:
If you mean the documentation, I've just made it as in neighbour functions. I can remove those "return" lines if it's critical